### ISSUE TITLE
Accessibility: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail

### ISSUE THREAD

**Issue by funkysi1701** [funkysi1701/funkysi1701.github.io#2823]
## Accessibility Issue Found by Pa11y

**WCAG Code**: `WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail`
**Total Occurrences**: 8
**URL**: https://blog-dev.funkysi1701.com
**Build**: 10.1.1.2333

---

### Example 1

**Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.39:1. Recommendation:  change background to #1363be.
**Selector**: `html > body > header > nav > div > h1 > a`

**Context:**
```html
<a class="navbar-brand flex-grow-1 flex-lg-grow-0 text-center text-lg-start mx-auto me-lg-3 active" href="https://blog-dev.funkysi1701.com:443/">Development Version of Funky Si...</a>
```

### Example 2

**Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.39:1. Recommendation:  change background to #1363be.
**Selector**: `#navbarSupportedContent > ul > li:nth-child(1) > a`

**Context:**
```html
<a class="nav-link" href="https://blog-dev.funkysi1701.com:443/about/">About</a>
```

### Example 3

**Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.39:1. Recommendation:  change background to #1363be.
**Selector**: `#navbarSupportedContent > ul > li:nth-child(2) > a`

**Context:**
```html
<a class="nav-link" href="https://blog-dev.funkysi1701.com:443/projects/">Projects</a>
```

---
_Detected by [Pa11y](https://pa11y.org/) accessibility checker in Azure Pipeline_
